.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+      - 'test'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
+
+      - name: Set secrets in .env file
+        run: |
+          cd ./src
+          echo "SLACK_SIGNING_SECRET=${{ secrets.SLACKSIGNINGSECRET }}" >> .env
+          echo "SLACK_OAUTH_TOKEN=${{ secrets.SLACKOAUTHTOKEN }}" >> .env
+          echo "SLACK_VERIFICATION_TOKEN=${{ secrets.SLACKVERIFICATIONTOKEN }}" >> .env
+          echo "OPENAI_API_KEY=${{ secrets.OPENAIKEY }}" >> .env
+          echo "PINECONE_INDEX=${{ secrets.PINCONEID }}" >> .env
+          echo "PINECONE_ENV=${{ secrets.PINECONEENV }}" >> .env
+          echo "SLACK_PORT=${{ secrets.SLACKPORT }}" >> .env
+          echo "SERPER_API_KEY=${{ secrets.SERPAAPIKEY }}" >> .env
+          echo "GITHUB_AUTH_TOKEN=${{ secrets.GITHUBOATHKEY }}" >> .env
+          echo "DAILY_TOKEN_LIMIT=${{ secrets.TOKENLIMIT }}" >> .env
+          echo "TEMPRATURE=${{ secrets.TEMPRATURE }}" >> .env
+
+        shell: bash
+
+
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install poetry
+          cd src
+          pip install numpy==1.25.2
```

Comment:
Doesn't `poetry install` ensure that `numpy` is installed?

Reply:
Please remove the explicit install of numpy here.  Running `poetry install` should install numpy for us.